packages/react/src/components/logged-in/index.tsx

```diff
@@ -27,7 +30,7 @@ export const LoggedIn: React.FC<Props> = ({ children, withFactorMethods }) => {
 
   const shouldRender = useMemo(() => {
     // user must be logged in
-    if (!isAuthenticated || !user) {
+    if (!isAuthenticated || !user || user.anonymous) {
```

Comment:
Makes sense to me! This might be a good candidate for a unit test

Reply:
I've actually changed my mind on this, it conflicts with the assumption made in `Groups`:
https://github.com/slashid/javascript/pull/234#issuecomment-2044794656

Reply:
This is a tough question and I was asking @balenio and @paulo-raca for help on determining this. We ended up on this being the definition of an authenticated user:

```
token is valid
person_type != "anonymous" (To defend from old token that had a missing field)
```

[Relevant discussion here](https://slashidworkspace.slack.com/archives/C048HJ7D5FD/p1710422233999519)

Reply:
Looks like we are aligned!

https://github.com/slashid/javascript/pull/234/files#r1559619822